docs/widgetbook-cloud/review.mdx

```diff
@@ -18,14 +18,6 @@ To use Widgetbook Cloud _Review_ your setup has to meet the following requiremen
 - Use the `widgetbook-cli` from the [widgetbook_cli](https://pub.dev/packages/widgetbook_cli) pub.dev package.
 - Setup a GitHub webhook
 
-## Setup the CLI
-
-For instructions on how to setup the CLI see [our docs](/widgetbook-cli/overview) or just run:
-
-```bash
-dart pub global activate widgetbook_cli
-```
-
```

Comment:
Do you think this information is redundant since it's already available at `/widgetbook-cli/overview`?

Reply:
Yes, I do also believe that it's redundant with the link to widgetbook_cli in the #requirements. I mainly removed it from this page because it isn't a required step for getting "Review" setup.

Reply:
Yep, I understand. Unfortunately, there is a problem with the docs page, that's why the code is not displayed correctly for the GitHub action. But I'll link an issue with the correct code now.